apps/opik-frontend/src/components/pages/CompareExperimentsPage/CompareExperimentsPanel/CompareExperimentsViewer.tsx

```diff
@@ -47,14 +47,16 @@ const CompareExperimentsViewer: React.FunctionComponent<
     }
   };
 
-  const renderContent = () => {
+  const renderOutput = () => {
     if (!isTraceExist) {
       return (
-        <NoData
-          title="No related trace found"
-          message="It looks like it was deleted or not created"
-          className="absolute inset-0 min-h-24"
-        />
+        <div className="mt-64">
```

Comment:
```suggestion
        <div className="my-auto">
```

Reply:
@andriidudar I didn't want to make it `my-auto` for the cases where we have a long dataset because it would look like 
<img width="1893" alt="image" src="https://github.com/user-attachments/assets/e4c346e1-e4de-46d8-904a-2fc36698de54">